src/component/AddTodoInput.js

```diff
@@ -0,0 +1,60 @@
+import '../App.css';
+import React, { Component } from 'react';
+import addImg from '../add.png';
+
+class AddTodoInput extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      todoContent: ''
+    };
+  }
+
+  render() {
+    return (
+      <div className='add-todo'>
+        <input
+          type='text'
+          placeholder='Add a to-do'
+          value={this.state.todoContent}
+          onKeyUp={this.checkEnterPressed}
+          onChange={this.updateStateContent}
+        />
+        <button
+          onClick={() => {
+            this.addTodo();
+          }}
+        >
+          <img src={addImg} alt='add' />
```

Comment:
Add title attribute to the img tag.
`<img src={addImg} alt='add' title={...} />`